libraries/types/dag_block/src/dag_block.cpp

```diff
@@ -37,24 +37,13 @@ DagBlock::DagBlock(string const &json)
       }()) {}
 
 DagBlock::DagBlock(Json::Value const &doc) {
-  if (auto const &v = doc["level"]; v.isString()) {
-    level_ = dev::jsToInt(v.asString());
-  } else {
-    // this was inconsistent with getJson()
-    // fixme: eliminate this branch
-    level_ = v.asUInt64();
-  }
+  level_ = dev::getUInt(doc["level"]);
```

Comment:
not sure where is this used but I am not a big fan of this silent return of some default value as we have no chance to notice that. Shouldnt all these fields be mandatory ?

Reply:
What do you mean by `silent return of some default value`? Where do you can see it?

Reply:
when there is no "level" in doc

Reply:
If we want to change the logic here and  check if we have all fields in json then we should create a separate task for it and change it in all places